articles/Introduction-to-time-series/index.md

```diff
@@ -0,0 +1,60 @@
+4. Time Series Forecasting methods
+5. Components of Time Series
+6. Conclusion
+
+Time series can be defined as a sequence or series of data points that are ordered in time. In traditional machine learning, the dataset is a collection of observations, and predictions are made based on the new data, which likely will not be known until the future. The future is what is predicted, but all the previous observations are put into considerations. In time series, the dataset is different. Time-series adds a definite order dependence between observations.
+
+Time series are assumed to be generated at a spaced timing. When the data in time series are timed are regular, they are called **regular time series** and when they are not at a regular interval, they are called **irregular time series**
+
+#### Real-life applications of Time series
+1. Can be used to predicting the closing price of the stock at the end of the day
+2. Used in Predicting the number of product sales
+3. Forecasting the birth or death rate at a hospital
+4. Forecasting the number of passengers a bus terminal will have 
+5. Used to forecast the unemployment rate of a city or state
+
+#### Time-series Forecasting
+Time series forecasting is fitting a model on a historical dataset, which is the training set, and then using it to predict a future occurrence, which is the test set. The performance of the time series model is rated based on its future prediction. The whole idea is that our model observes all the previous data and understands the foundational process in the series. Secondly, it predicts based on its understanding of the foundational process of the historical data.
+
+#### Time-series Analysis
+Time series analysis can be defined as getting meaningful insight and characteristics of the data to comprehend it better. Time series is beneficial in making predictions. It understands the scope of nature and uses it for future forecasting.
+
+#### Time Series Forecasting methods 
+Now let’s look at four time-series forecasting methods and have a brief description of them.
+1. Autoregressive Integrated Moving Average (ARIMA): This method combines both the Autoregression and Moving Average model. ARIMA models the next step in the sequence as a linear function based on the difference in observation and residual errors at previous steps. This method works best for univariate time series with trend and without seasonal components.
+2. Seasonal Autoregressive Integrated Moving-Average(SARIMA): the SARIMA models the next steps in a linear function style of the different observations, errors, differenced seasonal observation, and seasonal errors at the previous time steps. The SARIMA is fitted for univariate time series with the trend or seasonal components.
+3. Vector Autoregression (VAR): This method models the next step in each of the time series using the AR methods. It is the generalization of AR to multiple parallel time series e,g multivariate time series.
```

Comment:
Again, an introduction to AR is needed. Without the knowledge of AR, multiple parallel time series example does not make sense.